src/esp/bindings/GfxBindings.cpp

```diff
@@ -111,6 +113,32 @@ void initGfxBindings(py::module& m) {
 #endif
       .def("render_enter", &RenderTarget::renderEnter)
       .def("render_exit", &RenderTarget::renderExit);
+
+  py::enum_<LightPositionModel>(m, "LightPositionModel")
+      .value("CAMERA", LightPositionModel::CAMERA)
+      .value("GLOBAL", LightPositionModel::GLOBAL)
+      .value("OBJECT", LightPositionModel::OBJECT);
+
+  py::class_<LightInfo>(m, "LightInfo")
+      .def(py::init())
+      .def(py::init<Magnum::Vector3, Magnum::Color4, LightPositionModel>(),
+           "position"_a, "color"_a = Magnum::Color4{1},
+           "model"_a = LightPositionModel::GLOBAL)
+      .def_readwrite("position", &LightInfo::position)
+      .def_readwrite("color", &LightInfo::color)
+      .def_readwrite("model", &LightInfo::model)
+      .def("__eq__",
```

Comment:
```suggestion
      .def(py::self == py::self)
```

Same below. [Docs.](https://pybind11.readthedocs.io/en/stable/advanced/classes.html#operator-overloading). Tho you might need to [suppress some warnings](https://github.com/mosra/magnum-bindings/blob/15d0714d90ecb66957e8ab518c62ccf195b3945d/src/python/magnum/math.vector.h#L97-L100).